### PR TITLE
release 0.7.1

### PR DESCRIPTION
**Pull request by nsheff**
multi-server and build docs

- [x] wait for https://github.com/databio/refgenconf/pull/68
- [x] release date

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. 
 
+## [0.7.1] - 2019-10-23
+
+### Changed
+- `--genome_server` can now be called multiple times to add additional refgenieservers
```
Comment (michalstolarczyk) [inline docs/changelog.md]: let's change this to `--genome-server` to match the way we name multiple word arguments

File: docs/available_assets.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+```
+wget ftp://dbnsfp:dbnsfp@dbnsfp.softgenetics.com/dbNSFP4.0a.zip
+refgenie build test/dbnsfp --dbnsfp dbNSFP4.0a.zip
+```
+
+## Derived assets you can build
+
+For many of the following derived assets, you will need the corresponding software to build the asset.  You can either [install software on a case-by-case basis natively](build.md#install-building-software-natively), or you can [build the assets using `docker`](build.md#building-assets-with-docker).
+
+### bowtie2 index
```
Comment (michalstolarczyk) [inline docs/available_assets.md]: let's be consistent with the headers format here: "bowtie2 index" vs "ensembl_rb"

File: docs/changelog.md (hunk 1 above)
Comment (nsheff) [reply docs/changelog.md]: Agreed.

Comment (nsheff) [reply docs/changelog.md]: It actually is that way, it's just incorrect in the docs.

**Comment by nsheff** [comment]
> It feels like in the future we can leverage the fact that each section is identical structure-wise and make it a table. It'll make it even more transparent.

Exactly... in the future this will all be an automated output from refgenieserver for recipes.

**Comment by michalstolarczyk** [comment]
I think that both this and [refgenconf](https://github.com/databio/refgenconf/pull/68) are ready to go

**Pull request merged**